packages/calypso-products/src/main.ts

```diff
@@ -565,6 +565,11 @@ export function calculateMonthlyPrice( term: string, termPrice: number ): number
 	return parseFloat( ( termPrice / divisor ).toFixed( 2 ) );
 }
 
+export function calculateYearlyPrice( term: string, termPrice: number ): number {
+	const multiplier = getBillingMonthsForTerm( term );
+	return parseFloat( ( termPrice * multiplier ).toFixed( 2 ) );
```

Comment:
I'm not sure this function does what it says it's doing -- it seems to require passing in the combination of TERM_**ANNUALLY** for `term` and a **monthly** price for `termPrice` (kind of an odd combination) in order to work correctly.  If you pass in anything else, I don't think it will work as expected?

If you don't want to bother making it work generically, I think you could also just remove this function entirely and then have the calling code simply multiply by 12 (since the calling code is only working with monthly prices anyway).

Reply:
Fair enough.
It was meant as the mirror image of `calculateMonthlyPrice` by using multiplication instead of division.
To avoid delaying the launch of the experiment, we'll follow up on this in another PR.

Reply:
> I did notice it doesn't fix the issue for two-year plans (or presumably anything else besides monthly and yearly). Here's what it looks like for me with an $84 two-year-plan (the $78 is still incorrect):

Indeed, the fix has been limited to the scope of the Pricing test.
We can follow up with a fix to cover.